src/pages/CloudDeployProgress.js

```diff
@@ -95,14 +97,9 @@ class CloudDeployProgress extends BaseWizardPage {
   // need to go back
   resetPlaybookStatus = () => {
     if (this.props.playbookStatus) {
-      let playStatus = this.props.playbookStatus.slice();
-      playStatus.forEach((play, idx) => {
-        // remove the exist one
-        if (play.name === PRE_DEPLOYMENT_PLAYBOOK ||
-          play.name === DAYZERO_SITE_PLAYBOOK || play.name === SITE_PLAYBOOK) {
-          play.playId = '';
-          play.status = '';
-        }
+      // remove playbook status for any playbook in this.playbooks
+      let playStatus = this.props.playbookStatus.filter(status => {
+        return this.playbooks.indexOf(status.name) === -1;
```

Comment:
While this works, it is more intuitive and correct to use [includes()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) than comparing indexOf() with -1. For example:
```
let playStatus = this.props.playbookStatus.filter(status => ! this.playbooks.contains(status.name));
```